DS/DS8/SortingSystem.java

```diff
@@ -0,0 +1,92 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+public class SortingSystem {
+	static int[] list=null;
```

Comment:
ACTION: In java class level reference  variables are by default initialized with null, you don't have to explicitly initialize them.